controllers/reconciler_common.go

```diff
@@ -31,16 +31,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+// ReconcilerCommon -
 type ReconcilerCommon interface {
 	GetClient() client.Client
 	GetLogger() logr.Logger
 }
```

Comment:
Perhaps the content of this `controllers/reconciler_common.go` file is now outdated by equivalent newer options in `lib-common`?

Reply:
Yes likely, this module will be cleaned up later with other resources.